app/tests/integration/nimbus/test_desktop_client_integrations.py

```diff
@@ -190,3 +197,179 @@ def test_check_telemetry_enrollment_unenrollment(
+        experiment_slug,
+        "desktop",
+        targeting,
+        data,
+    )
+    wait = WebDriverWait(selenium, 10)
+    selenium.get("about:config")
+    search_bar = wait.until(EC.presence_of_element_located(_search_bar_locator))
+    search_bar.send_keys("nimbus.qa.pref-1")
+    wait.until(EC.presence_of_element_located(_row_locator))
+    elements = selenium.find_elements(*_row_locator)
+    for item in elements:
+        if "default" in item.text:
+            assert True
+        else:
+            continue
+    summary = SummaryPage(selenium, urljoin(base_url, experiment_slug)).open()
+    summary.launch_and_approve()
+
+    kinto_client.approve()
+
+    summary = SummaryPage(selenium, urljoin(base_url, experiment_slug)).open()
+    summary.wait_for_live_status()
+
+    # Ping the server twice as it sleeps sometimes
+    requests.get("http://ping-server:5000/pings")
+    time.sleep(5)
+
+    # Check their was a telemetry event for the enrollment
+    control = True
+    timeout = time.time() + 60 * 5
+    while control or time.time() < timeout:
+        telemetry = requests.get("http://ping-server:5000/pings").json()
+        for item in reversed(telemetry):
+            if "events" in item["payload"]:
+                if "parent" in item["payload"]["events"]:
+                    for events in item["payload"]["events"]["parent"]:
+                        try:
+                            assert "normandy" in events
+                            assert "enroll" in events
+                            assert "nimbus_experiment" in events
+                            assert experiment_slug in events
+                        except AssertionError:
+                            continue
+                        else:
+                            control = False
+                            break
+            else:
+                continue
+        # If there are no pings we have to wait
+        else:
+            trigger_experiment_loader()
+            time.sleep(15)
+    else:
+        if control is not False:
+            assert False, "Experiment enrollment was never seen in ping Data"
+
+    # check experiment exists, this means it is enrolled
+    for item in requests.get("http://ping-server:5000/pings").json():
+        if "experiments" in item["environment"]:
+            for key in item["environment"]["experiments"]:
+                if experiment_slug in key:
+                    break
+                else:
+                    continue
+
+    selenium.get("about:config")
+    search_bar = wait.until(EC.presence_of_element_located(_search_bar_locator))
+    search_bar.send_keys("nimbus.qa.pref-1")
+    wait.until(EC.presence_of_element_located(_row_locator))
+    elements = selenium.find_elements(*_row_locator)
+    for item in elements:
+        if "test_string_automation" in item.text:
+            assert True
+            break
+        else:
+            continue
+    else:
+        assert False
```

Comment:
Ya see if we break the about:config into a Page object then we can reuse these lookups.

Reply:
Thing is, we won't be doing anything with about:config outside of these tests. Is it worth adding a whole page model for just 1 test?